Nfield.Quota/QuotaLevelDefinition.cs

```diff
@@ -9,7 +9,7 @@ public class QuotaLevelDefinition : IEquatable<QuotaLevelDefinition>
 
         public static bool operator ==(QuotaLevelDefinition left, QuotaLevelDefinition right)
         {
-            return left?.Equals(right) ?? false;
+            return left?.Equals(right) ?? ReferenceEquals(right, null);
```

Comment:
if `ReferenceEquals` returns true do you even need to call the other one anymore? Maybe the order should be reversed?

Reply:
You would prefer:
return ReferenceEquals(left, right) ? true : left?.Equals(right) ?? false;
?
Maybe faster but less readable I think.

Reply:
Yeah, you are right. The nullability is annoying here. I was thinking `ReferenceEquals(left, right) || left?.Equals(right)` but that doesn't work with nulls.